desktop/assets/texts/en-US/main_menu.ftl

```diff
@@ -27,6 +27,10 @@ help-menu-sponsor-development = Sponsor Development...
 help-menu-translate-ruffle = Translate Ruffle...
 help-menu-about = About Ruffle
 
+bookmarks-menu = Bookmarks
+bookmarks-menu-add = Add
```

Comment:
> Wasn't sure what this option did at first, but it looks like it adds the current playing movie as a bookmark right?

Yes.

> I think, although it feels a bit of a step backwards right now (sorry) and maybe lots more work (double sorry), can we go with "Add..." and have it pop up a dialog for creating a bookmark - defaulting the name and location? We can reuse that dialog for editing bookmarks, and in the future use it for all the per-bookmark settings too.

I mostly designed the current dialog on how the Firefox web browser does it, I guess the browser has a little popup allowing you to change the name, we don't. We could add an initial dialog to add a bookmark where the user can cancel + change the name. Just in my opinion, player options feels too advanced to just have in the initial dialog.

Reply:
That's totally fine - I just know that "Add" is going to confuse people and they'll click on it to see what it does, and then not know what it did because there's no feedback.
I can't think of something to rename it to ("Bookmark Current Movie" feels too clunky, anything else feels too nondescript) and I can't think of any feedback, so a popup of some kind feels best.

Reply:
I feel like if we are going the popup/dialog route for add button, then it should always be available and default name/location to the current movie if there is one.

Reply:
That'd be nice!

Reply:
Added the dialog/popup.